Src/WitsmlExplorer.Frontend/.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_WE_URL=https://localhost/witsmlexplorer
+NEXT_PUBLIC_WE_API_URL=https://localhost/witsmlexplorer.api
```

Comment:
With this PR, backend is not modified, backend can run on any host/port/path. Eg. https://localhost:5555;
Then we set NEXT_PUBLIC_WE_API_URL =  https://localhost:5555

When the frontend prepends 'api' then it would be like https://localhost:5555/api/witsml-servers

Versioning API won't change this implementation, as we are parameterizing only host/port/path. When we introduce versioning of API then URL will look like  https://localhost:5555/api/v1/witsml-servers, which needs to be handled in backend/frontend code

Reply:
Awaits merge of this PR until documentation is provided how to setup local environment due this change by an pr on issue #354.

Reply:
All the steps are documented and created a PR https://github.com/equinor/witsml-explorer/pull/385